src/backend/launcher.ts

```diff
@@ -121,16 +124,32 @@ async function prepareLaunch(
     gameSettings.useSteamRuntime &&
     (isNative || gameSettings.wineVersion.type === 'proton')
   if (shouldUseRuntime) {
+    let nonNativeRuntime: SteamRuntime['type'] = 'soldier'
+    if (!isNative) {
+      try {
+        const parentPath = normalize(join(gameSettings.wineVersion.bin, '..'))
+        const requiredAppId = VDF.parse(
+          readFileSync(join(parentPath, 'toolmanifest.vdf'), 'utf-8')
+        )['manifest']['require_tool_appid']
+        if (requiredAppId === 1628350) nonNativeRuntime = 'sniper'
+      } catch (error) {
+        logError('Failed to parse toolmanifest.vdf', LogPrefix.Backend)
```

Comment:
Logging the error if something goes wrong is generally a good idea. E.g.:
```suggestion
        logError(['Failed to parse toolmanifest.vdf:', error], LogPrefix.Backend)
```